.github/workflows/ci.yml

```diff
@@ -67,9 +70,17 @@ jobs:
 
     - name: Verify Pacts
       run: |
+        export IS_MERGED=${{ github.event.pull_request.merged }}
         export PACT_BROKER_BASE_URL='https://edx.pactflow.io'
         export PACT_BROKER_TOKEN=${{ secrets.PACT_FLOW_ACCESS_TOKEN }}
         export PUBLISH_VERSION=`git rev-parse --short HEAD`
-        export PUBLISH_TAGS=${{ github.head_ref }}
+        if [ $IS_MERGED == false ]
+        then
+          export PUBLISH_TAGS=${{ github.head_ref }}
+          export GIT_ENV='development'
+        else
+          export PUBLISH_TAGS=${GITHUB_REF:11}
```

Comment:
Why can't we use `github.ref` directly here?

Reply:
Same.

Reply:
oh, it looks like we can use $GITHUB_REF directly as well. https://docs.github.com/en/actions/learn-github-actions/environment-variables

Removing the above declaration.